kdc/windc_plugin.h

```diff
@@ -74,7 +81,7 @@ typedef krb5_error_code
 	KDC_REQ *, METHOD_DATA *);
 
 
-#define KRB5_WINDC_PLUGIN_MINOR			6
+#define KRB5_WINDC_PLUGIN_MINOR			7
```

Comment:
Right, you've changed the ABI version so, you can change the ABI, right?

Reply:
Done.